packages/cli/git/config/.gitconfig

```diff
@@ -22,3 +22,5 @@
 	tool = opendiff
 [help]
 	autocorrect = 1
+[credential]
+	helper = /usr/lib/git-core/git-credential-gnome-keyring
```

Comment:
It makes gnome keyring store git credentials. I'm taking it out because ssh keys are far better